db/migrate/20230925190004_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Hello! In accordance with project requirements, kindly modify the reference from "user" to "author" for the post. This adjustment will generate an author_id instead of a user_id. Additionally, please ensure that in the foreign key relationship, you explicitly specify the connection to the "users" table. 🔄💻 Thank you!